examples/this-that/wasm/src/lib.rs

```diff
@@ -0,0 +1,30 @@
+#![no_std]
+
+use gstd::msg;
+use parity_scale_codec::Encode;
+use this_that_app::{commands, queries};
+
+// TODO: This function might be generated by some macro like `gprogram`
+#[no_mangle]
+extern "C" fn handle() {
+    let command = msg::load::<commands::Commands>()
+        .expect("This needs to be handled in a consistent way: input parse error");
+    let (result, is_error) = commands::handlers::handle_commands(command);
```

Comment:
It is more idiomatic not to include a module for type and include only one level to function (https://doc.rust-lang.org/book/ch07-04-bringing-paths-into-scope-with-the-use-keyword.html#creating-idiomatic-use-paths).
```suggestion
    let command: Commands = msg::load()
        .expect("This needs to be handled in a consistent way: input parse error");
    let (result, is_error) = handlers::handle_commands(command);
```